tools/inference_engine/src/inference_engine_operators.cpp

```diff
@@ -27,12 +28,22 @@ INFERENCE_ENGINE_API inference_engine_result_t inferenceEngineSetResource(infere
     return INFERENCE_ENGINE_RESULT_SUCCESS;
 }
 
+INFERENCE_ENGINE_API inference_engine_result_t inferenceEngineCheckNodeInputs(inference_engine_node_t node)
+{
+	std::cout << "inferenceEngineCheckNodeInputs" << std::endl;
+	auto typed_node = reinterpret_cast<inference_engine::INode*>(node);
+	if (typed_node->check_inputs())
```

Comment:
Typed node should probably expose more generic name like for example "validate()" (or it should be done in c-tor),  checking inputs is an implementation detail of each node